pkg/boot/bls/bls.go

```diff
@@ -195,6 +197,9 @@ func parseLinuxImage(vals map[string]string, fsRoot string) (boot.OSImage, error
 	// If both title and version were empty, so will this.
 	linux.Name = strings.Join(name, " ")
 	linux.Cmdline = strings.Join(cmdlines, " ")
+	if len(initrds) > 0 {
+		linux.Initrd = boot.CatInitrds(initrds...)
```

Comment:
I'm wondering if it's preferable to only use CatInitrds if there's more than one initrd. Or maybe CatInitrds should return the first io.ReaderAt if there's only one?

Reply:
I think the latter makes CatInitrds somewhat more self-contained so that you wouldn't need to have that extra logic. Currently it would just have some overhead when copying only one initrd. Makes sense. :)

Reply:
This still needs to happen.